tools/on_device_tests_gateway_client.py

```diff
@@ -222,6 +224,18 @@ def main():
       help='Optional argument to specify which unit testing shard to run in '
       'the On-Device Tests Job. Defaults behavior is to run all tests without '
       'sharding enabled.')
+  trigger_parser.add_argument(
+      '--test_attempts',
+      type=str,
+      default='1',
+      required=False,
+      help='The maximum number of times a test could retry.')
+  trigger_parser.add_argument(
+      '--retry_level',
+      type=str,
+      default='ERROR',
+      required=False,
+      help='The retry level of Mobile harness job. Either ERROR or FAIL.')
```

Comment:
Could we leave a bit more context here as to what ERROR or FAIL mean ? I mean i can look them up on the doc page separately, but to a casual reader they sort of look similar. tl;dr i want to ensure we only retry when we get infra errors, not when we get actual distinguishable test failures